### ISSUE TITLE
Double catch caps in HCRs?

### ISSUE THREAD

**Issue by rwildermuth** [futureseas/SardineMSE#6]
@detommas I was checking the way these HCRs are implemented and found that in some cases two catch caps are implemented, e.g.:
https://github.com/futureseas/SardineMSE/blob/6363863ad5c15b3d00267a5f34e5c93a7ea39d7e/R/MS_sar_hcr5_018.R#L168

In this case, the Pikitch rule defines a catch cap at 0.8B0, but then if biomass is high enough the current MAXCAT 200000 t cap is applied on top of the 0.8B0 cap. Is this intended or should the MAXCAT cap be removed (i.e., remove the line in the code linked above)?

**Comment by detommas** [comment]
Yes, having the 200,000 catch cap is intended, so that the only difference
between the Pikitch rule and the current one is the F target. 0.8B0 is not
associated with a catch cap, but with an F cap (if bio is larger than 0.8B0
F=Fmsy, if lower F decreases linearly with biomass until 0.4B0 is reached).
